index.html

```diff
@@ -0,0 +1,139 @@
+        <div class="join"><a href="#form">Join</a></div>
+        <div class="contact" > <a href="#contact">Contact us </a></div>
+        </div>
+    </nav>
+  
+
+    <section class="intro-section">
+
+      <article class="intro-align">
+    <div class="crypto-text" id="intro">"Blockchain crypto!"</div>
+
+    
+    <h1 class="ico-title">NEW CRYPTO COIN 2022 INITIAL COIN OFFERING</h1>
+    <div class="ico-words"> The crypto currency that has come to create financial freedom and be the foundation of the future of finance,all over the world. The ico will be taking place on December .</div>
+    <h2 class="ico-date">2022.12.01(TUE)~30(FRI)</h2>
+    <div id="contact" class="contact">@ www.fakenewcryptocoin.com</div>
+      </article>
+
+    </section>
+
+
+    <section class="main-section">
+        
+    <h2 class="program-class">Main Program</h2>
+        
+        <div class="card-flex">
+            
+      <div class="card">
+        <img src="./images/download.jpeg" alt="whitepaper image" class="program-img">
+        <div class="card-container">
+          <h4><b>Release of official complete white paper</b></h4>
+        </div>
+
+      </div>
+
+      <div class="card">
+        <img src="./images/download.png" alt="explanation" class="program-img">
+        <div class="card-container">
+          <h4><b>Explanation of whitepaper</b></h4>
+        </div>
+      </div>
+      
+        <div class="card">
+            <img src="./images/download (1).jpeg" alt="roadmap" class="program-img">
+            <div class="card-container">
+              <h4><b>official roadmap</b></h4>
+            </div>
+        
+        </div>
+      </div>
+        </div>
+        
+        <form action="https://formspree.io/f/xbjbawkn" method="post" id="form">
+            <input type="text" name="Name" maxlength="30" placeholder="Name">
+    
+            <div class="submit">
+            Please type/text your email here to receive authentication email to participate in the ICO.
+            <textarea name="e-mail" cols="40" rows="1" placeholder="E-mail"></textarea>
+            <button type="submit" value="Submit">Submit</button>
+          </div>
+        </form> 
+    </section>
+
+    
+
+    <section class="speaker-section">
+        
+        <ul class="speakers-list">
+    <h2>Featured Speakers</h2>
+    <hr class="speaker-line1">
+
+    <div class="list-container">
+    
+       
+    </div>
+
+    </div>
+    </ul>
+    
+    <div class="more-button"><button class="more-width">MORE</button></div>
+    
+    </section> 
+
+    <section>
+        <div class="partners-class">
+            <h3 class="partners">Partners</h3>
+            <hr class="speaker-line1" />
+            <ul class="sponsors-list">
+              <li>Mozilla</li>
+              <li>Google</li>
+              <li>NAVER</li>
+              <li>daumkakao</li>
+              <li>airbnb</li>
+            </ul>
+          </div>
```

Comment:
The size of the Sponsors is too large, and the `flex-wrap:wrap` is not working. 
| Your Design   | Behance Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219805160-fa543d4c-6253-4f4e-bc68-c80f8dae9a7e.png)| ![image](https://user-images.githubusercontent.com/61361037/219805133-6d034432-2e46-4778-8aa9-608020fe54e5.png)|

I have tested out the following code for `.sponsors-list`, this might improve the design.
```
display: flex;
flex-direction: row;
list-style: none;
flex-wrap: wrap;
justify-content: space-between;
font-size: 38px;
padding: 0 40px;
```

Reply:
@RaoAkif thanks for your suggestion